solutions/src/main/java/ru/sberbank/school/task02/CurrencyCalc.java

```diff
@@ -0,0 +1,36 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class CurrencyCalc implements FxConversionService {
+
+    private ExternalQuotesService externalQuotesService;
+
+    public CurrencyCalc( ExternalQuotesService externalQuotesService) {
+        this.externalQuotesService = externalQuotesService;
+    }
+
+
+    @Override
+    public BigDecimal
+  convert( ClientOperation operation, Symbol symbol, BigDecimal amount) {
```

Comment:
**Reporter**: Checkstyle  **Rule**: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck **Severity**: WARN
&#39;convert&#39; has incorrect indentation level 2, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-2088256065>*